src/main/webapp/go.html

```diff
@@ -4,19 +4,40 @@
     <meta charset="UTF-8" />
     <title>Go On Scavenger Hunt</title>
     <link rel="stylesheet" href="go.css" />
+    <script src="config.js"></script>
     <script src="go.js"></script>
   </head>
   <body>
     <div class="content">
+      <div class="center-block">
+        <div id="map-area"></div>
+      </div>
+      <div class="center-row">
+        <div id="map-message-area"></div>
+      </div>
       <form action="/name-data" method="POST">
-        <input type="text" name="name-input" /><br/>
-        <input type="submit" /><br /><br />
+        <input type="text" name="name-input" size="50"/>
+        <input type="submit" class="button" id="submit-button"/>
```

Comment:
Is the button class going to be used for something in the future?

Reply:
And in general, I'm seeing classes that aren't currently being styled in the css file. Are they supposed to be in there or something?

Reply:
These additional classes accidentally incorporated some of the additions I've made for a future pull request -- I'll remove these additions for this PR.